plugin-server/src/main/ingestion-queues/session-recording/session-recordings-blob-consumer.ts

```diff
@@ -267,33 +257,35 @@ export class SessionRecordingBlobIngester {
             },
 
             team_id: teamId,
-            distinct_id: event.distinct_id,
+            distinct_id: event.properties.distinct_id,
```

Comment:
is that change right?! Would mean we've not had distinct id all this time?

In which case then we don't need it?

Reply:
Indeed! - it never worked :D But we never used until now as it was just writing stuff to S3 so the distinctId was never referenced. Now we need it for the replay events which is why I corrected it